src/components/DateRangePicker.jsx

```diff
@@ -228,9 +228,9 @@ export default class DateRangePicker extends React.Component {
 
   doesNotMeetMinimumNights(day) {
     const { startDate, focusedInput, minimumNights } = this.props;
-    const today = moment().startOf('day');
-    const dayDiff = startDate ? day.diff(startDate, 'days') : day.diff(today, 'days');
+    if (!startDate) return false;
 
+    const dayDiff = day.diff(startDate, 'days');
```

Comment:
Mmm fair point. Given that we allow blocked days inside ranges right now,
what you're saying is true. So yes, probably we'd want
`this.props.isOutsideRange` there.

On Wed, Aug 24, 2016, 10:22 PM Dhananjoy Biswas <notifications@github.com>
wrote:

> In src/components/DateRangePicker.jsx
> <https://github.com/airbnb/react-dates/pull/12#discussion_r76181681>:
>
> >
> > +    const dayDiff = day.diff(startDate, 'days');
>
> doesNotMeetMinimumNights =
> this.isBlocked(moment(day).subtract(minimumNights, 'days'));
>
> @majapw <https://github.com/majapw> did you mean doesNotMeetMinimumNights
> = this.props.isOutsideRange(moment(day).subtract(minimumNights, 'days'));?
> Because moment(day).subtract(minimumNights, 'days') can be blocked for
> other reasons, in those case it is possible to select day as end date and
> some other non-blocked day before moment(day).subtract(minimumNights,
> 'days') as start date.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/airbnb/react-dates/pull/12/files/da420d1fe4bce2d4d4d12cabfc773d24a81525a6#r76181681>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABUdtVKuuPrpKaP49QlJDA6gIr-jwefMks5qjSaBgaJpZM4JnwPM>
> .
>